internal/pkg/sylog/sylog.go

```diff
@@ -158,6 +158,17 @@ func Debugf(format string, a ...interface{}) {
 // SetLevel explicitly sets the loggerLevel
 func SetLevel(l int) {
 	loggerLevel = messageLevel(l)
+	// set the apex log level, for umoci
+	if loggerLevel <= log {
+		// quiet or silent options
```

Comment:
I still don't think we want to do this for quiet a warning isn't normal output.

`-q, --quiet     suppress normal output`